config/endpoints/4_ministere_interieur_v4.yml

```
@@ -97,54 +97,53 @@
     - Portail GRU - Instruction des démarches
   use_cases_forbidden:
     - Portail GRU - Préremplissage
-  parameters:
+  parameters: &association_parameters
     - Numéro de SIREN ou numéro RNA de l’association
   faq: &association_faq
     - q: Une association utilisatrice de votre démarche s'est aperçue que ses informations ne sont plus à jour ?
       a: |+
         Vous pouvez l'orienter vers le [site officiel Le compte Asso](https://lecompteasso.associations.gouv.fr/declarer-un-changement-de-situation-de-mon-association/){:target="_blank"} où elle pourra déclarer son changement de situation.
 
-# - uid: 'ministere_interieur/unites_legales_open_data'
-#   path: '/v4/ministere_interieur/api-asso/associations/open_data/{siren_or_rna}'
-#   position: 150
-#   perimeter: *association_perimeter
-#   call_id: "SIREN / RNA"
-#   provider_uids: *association_provider_uids
-#   keywords: &association_keywords
-#   data:
-#     description: |+
-#       {:.fr-highlight.fr-highlight--caution}
-#       ⚠️ **Le type et la quantité d'informations disponibles par association est variable** car de nombreuses ressources sont déclaratives ou déposées par les associations dans le cadre de démarches en ligne.
+- uid: 'ministere_interieur/unites_legales_open_data'
```

Changer l'URL pour uniformiser avec l'existant (https://entreprise.api.gouv.fr/catalogue/ministere_interieur/associations) :
```suggestion
- uid: 'ministere_interieur/associations_open_data'
```